R/project.R

```diff
@@ -1,4 +1,8 @@
 
+# The path to the currently-loaded project, if any.
+# NULL when no project is currently loaded.
+`_renv_project_path` <- NULL
```

Comment:
I don't understand why this works, because I thought the namespace locked all bindings when the package was loaded. To get around that problem, we usually do something like:

```R
the <- new.environment(parent = emptyenv)
the$project_path <- NULL
```

But I don't understand why it's not affecting you here

Reply:
Ooooh, because you specifically unlock everything starting with `_` in `renv_binding_init()`

Reply:
Another one of `renv`s dirty tricks. 🙂